src/collection/components/CollectionOrBundleEdit.tsx

```diff
@@ -1307,30 +1316,32 @@ const CollectionOrBundleEdit: FunctionComponent<
 			...createDropdownMenuItem(
 				'save',
 				tText('collection/views/collection-edit___opslaan'),
-				'download',
+				IconName.download,
 				true
 			),
 			...createDropdownMenuItem(
 				'openPublishModal',
-				tText('collection/components/collection-or-bundle-edit___delen'),
-				'share-2',
+				isPublic ? tText('Maak privé') : tText('Publiceer'),
+				isPublic ? IconName.unlock3 : IconName.lock,
 				true
 			),
 			...createDropdownMenuItem(
 				'redirectToDetail',
 				tText('collection/components/collection-or-bundle-edit___bekijk'),
-				'eye',
+				IconName.eye,
 				true
 			),
 			...createDropdownMenuItem(
 				'rename',
 				isCollection
 					? 'Collectie hernoemen'
 					: tText('collection/components/collection-or-bundle-edit___bundel-hernoemen'),
-				'folder',
+				IconName.folder,
 				true
 			),
-			...createDropdownMenuItem('delete', 'Verwijderen', 'delete', true),
+			...createDropdownMenuItem('delete', tText('Verwijderen'), IconName.delete, true),
+
+			...createDropdownMenuItem('share', tText('Delen'), IconName.userGroup, isCollection),
```

Comment:
This seems more logical if it is located close to the publish option, so it has a similar order to the desktop page.